lib/daru/index/index.rb

```diff
@@ -44,8 +44,9 @@ def each(&block)
     end
 
     attr_reader :relation_hash, :size
+    attr_accessor :name
 
-    def initialize index
+    def initialize index, opts={} # rubocop:disable Metrics/MethodLength
```

Comment:
@zverok , are you saying to define separate method for [L50](https://github.com/SciRuby/daru/pull/323/files#diff-a1adf8860e167030bf548e2529c7d508R50) to [L61](https://github.com/SciRuby/daru/pull/323/files#diff-a1adf8860e167030bf548e2529c7d508R61), to maintain the MethodLength ?